episodes/04-geo-landscape.md

```diff
@@ -112,15 +114,15 @@ Benefits of using a GUI include:
 
 Downsides of using a GUI include:
 
-  - Low reproducibility - you can't record your actions and replay
-  - Most are not designed for batch-processing files
+  - Low reproducibility - you can record your actions and replay, but this requires some knowledge of the software
+  - Batch-processing is possible, but requires knowledge of the software 
```

Comment:
By saying this, to me it does not sound like a "downside" anymore, since you can argue more knowledge is needed using Python. Maybe we can emphasize the reproducibility and Batch-processing is limited comparing to scripting?